OverlayPlugin.Core/EventSources/MiniParseEventSource.cs

```diff
@@ -279,7 +279,7 @@ private void InitFFXIVIntegration()
                     {
                         foreach (var name in names)
                         {
-                            if (combatantName.Equals(name))
+                            if (combatantName.Equals(name, StringComparison.CurrentCultureIgnoreCase))
```

Comment:
You need to add a ToString() to the combatantName, possibly where it's declared above.

Reply:
Yeah, with that change, it builds and works for me locally.